src/Microsoft.AspNet.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -15,14 +15,15 @@
 namespace Microsoft.AspNet.JsonPatch.Adapters
 {
     /// <inheritdoc />
-    public class ObjectAdapter<TModel> : IObjectAdapter<TModel> where TModel : class
+    internal class ObjectAdapter : IObjectAdapter
```

Comment:
Hi Kirthi, I don't think there's a real use case for a consumer of the component to be able to use the IObjectAdapter implementation we provide outside of this component.  Either the consumer uses ApplyTo, which instantiates the ObjectAdapter, or he can provide his own implementation of an IObjectAdapter if he chooses to.